libflux/flux-core/src/parser/tests.rs

```diff
@@ -3147,6 +3147,16 @@ fn test_parse_record_type_tvar_properties() {
     )
 }
 
+#[test]
+fn test_parse_record_unclosed_error() {
```

Comment:
I believe this test should check the behaviour? @jsternberg

Reply:
Oh github collapsed this change so I didn't see it.